opm/autodiff/MissingFeatures.cpp

```diff
@@ -22,12 +22,30 @@
 #include <opm/autodiff/MissingFeatures.hpp>
 #include <unordered_set>
 #include <string>
+#include <map>
 
 
 namespace Opm {
 
 namespace MissingFeatures {
 
+    template<typename T>
+    struct PartiallySupported {
+        int item;
```

Comment:
Yeah, that's fine. My concern is that in the json format for these keywords, the options short cut(that is option name) are somehow different to Eclipse Manual, therefor I choose number instead, if you still think the item name is better, I will change.